camel/types/enums.py

```diff
@@ -43,34 +43,20 @@ def value_for_tiktoken(self) -> str:
 
     @property
     def is_openai(self) -> bool:
-        r"""Returns whether this type of models is an OpenAI-released model.
-
-        Returns:
-            bool: Whether this type of models belongs to OpenAI.
-        """
-        if self.name in {
-                "GPT_3_5_TURBO",
-                "GPT_3_5_TURBO_16K",
-                "GPT_4",
-                "GPT_4_32K",
-                "GPT_4_TURBO",
-                "GPT_4_TURBO_VISION",
-        }:
-            return True
-        else:
-            return False
+        r"""Returns whether this type of models is an OpenAI-released model."""
+        return self.name in {
+            "GPT_3_5_TURBO",
+            "GPT_3_5_TURBO_16K",
+            "GPT_4",
+            "GPT_4_32K",
+            "GPT_4_TURBO",
+            "GPT_4_TURBO_VISION",
```

Comment:
We can also fix it:
```suggestion
        return self in {
            ModelType.GPT_3_5_TURBO,
            ModelType.GPT_3_5_TURBO_16K,
            ModelType.GPT_4,
            ModelType.GPT_4_32K,
            ModelType.GPT_4_TURBO,
            ModelType.GPT_4_TURBO_VISION,
```
we can also fix `is_open_source` and other places that use `name`.